build/build_edge_deb.sh

```diff
@@ -82,6 +85,18 @@ THIS_SCRIPT_DIR=$(dirname "$THIS_SCRIPT")
 XRT_DIR=`readlink -f $THIS_SCRIPT_DIR/../`
 DEBIAN=`readlink -f $THIS_SCRIPT_DIR/debian`
 
+if [ -z $DIST ]; then
+    error "-dist is required option"
+fi
+
+if [[ $DIST == "jammy" ]]; then
+    OS_VERSION="22.04"
+elif [[ $DIST == "focal" ]]; then
+    OS_VERSION="20.04"
```

Comment:
All this looks very ad-hoc and we should use some Debian/Ubuntu recipe for this.